go.mod

```diff
@@ -3,22 +3,24 @@ module github.com/owncloud/ocis-graph
 go 1.13
 
 require (
-	contrib.go.opencensus.io/exporter/jaeger v0.2.0
-	contrib.go.opencensus.io/exporter/ocagent v0.6.0
-	contrib.go.opencensus.io/exporter/zipkin v0.1.1
-	github.com/cs3org/go-cs3apis v0.0.0-20191128165347-19746c015c83
-	github.com/cs3org/reva v0.0.2-0.20200115110931-4c7513415ec5
-	github.com/go-chi/chi v4.0.2+incompatible
+	contrib.go.opencensus.io/exporter/jaeger v0.2.1
+	contrib.go.opencensus.io/exporter/ocagent v0.7.0
+	contrib.go.opencensus.io/exporter/zipkin v0.1.2
+	github.com/cs3org/go-cs3apis v0.0.0-20200730121022-c4f3d4f7ddfd
+	github.com/cs3org/reva v1.1.1-0.20200812082732-2f9d60a99d86
```

Comment:
does it also work with `v1.1.0`? It'd be better to have a released version than the current master at this point on time

Reply:
using v1.1.0 now